packages/components/src/ButtonGroup.tsx

```diff
@@ -0,0 +1,31 @@
+import type { ButtonGroupVariants } from './styles/ButtonGroup.css';
+import type { ComponentPropsWithRef, ForwardedRef } from 'react';
+
+import { clsx } from 'clsx';
+import { forwardRef } from 'react';
+import { ButtonContext } from 'react-aria-components';
+
+import { variants } from './styles/ButtonGroup.css';
+
+type ButtonGroupProps = ComponentPropsWithRef<'div'> &
+  ButtonGroupVariants & {
+    isDisabled?: boolean;
+  };
+
+const ButtonGroup = forwardRef(
```

Comment:
Curious: should we carry this group component forward?

The RAC-based components represent our pristine future, and IIRC ButtonGroup as it is today isn't one we're in love with.

Reply:
I'd say yes if only for the styles for grouping them (which have been cleaned up) as it is a pattern we'd want consistency around. They even [have an example](https://react-spectrum.adobe.com/react-aria/Button.html#advanced-customization) for one. Can always drop it if we want to move a different direction later.

Reply:
Oh I missed you built the grouping on top of RAC!

Ignore me.